torchao/quantization/quant_primitives.py

```diff
@@ -323,21 +323,22 @@ def dynamically_quantize_per_tensor(
 
 
 def quantize_activation_per_token_absmax(t):
-    n_bits = 8
     # if the shape of t is [B, N, K], the shape of scales will be [B, N, 1]
+    mapping_type = MappingType.SYMMETRIC
+    block_size = list(t.shape)
+    for i in range(len(block_size) - 1):
+        block_size[i] = 1
+    dtype = torch.int8
+    scale, zero_point = choose_qparams_affine(t, mapping_type, block_size, dtype, scale_dtype=torch.float)
 
-    scales = t.abs().amax(dim=-1, keepdim=True)
-    if scales.dtype == torch.float16:
-        scales = (
-            scales.float()
-        )  # want float scales to avoid overflows for fp16, (bf16 has wide enough range)
-    q_max = 2 ** (n_bits - 1) - 1
-    scales = scales.clamp(min=1e-5).div(q_max)
```

Comment:
Is this implicit now?

Reply:
Where did the `1e-5` go?

Reply:
oh sorry, I need to add `eps` for this one